src/exchange-connector/resourceLoader/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+export { CommerceStandardsLoader } from "./commerceStandardsLoader";
```

Comment:
I don't like this name.  It is too specific to something that should be broader.  Can we call it `FatRamlResourceLoader` instead pleasE?

Reply:
What do you mean subsequent PR?  Why split it?

This change changes what the interface is and breaking changes like that I would prefer to keep to a single PR

Reply:
just want to have the name change review easier... Anyways changed it in this PR itself...